### PR TITLE
Finalize science content

### PR DESCRIPTION
**Pull request by davidlmobley**
- [x] Adjust issues with last edit to aims
- [x] Read/check Aims one last time
- [x] Fix some stuff in Timeline
- [x] Add a sentence or two to proposal on backup plan if D3R ends
- [x] Standardize format of Outlook, Management plan with other sections
- [x] Read/check proposal one more time

### CODE REVIEW

**Comment by davidlmobley** [comment]
@jchodera - I think it's done. Skim and approve? Or OK with it now? (very minor changes here.)

File: latex/Research_Strategy.tex [hunk 1]
```diff
@@ -356,10 +356,10 @@
 % Sustained funding will allow us to do that. 
 % (This will also help answer the argument that we don't need funding for this...)
 \textbf{This new experimental data is critical to maximize impact on the modeling community.} 
-While the community has already benefitted considerably from SAMPL, as indicated by the nearly 100 publications on SAMPL (see attached list), the citation count, and lessons learned, the benefits are not what they could have been if the effort had funding.
+While the community has already benefitted considerably from SAMPL, as indicated by the nearly 100 publications on SAMPL (see attached list), the citation count, and the growing participation (Figure~\ref{figure:sampl_history}), the benefits are not what they could have been if the effort had funding.
```
Comment (jchodera) [inline latex/Research_Strategy.tex]: > the benefits are not what they could have been if the effort had funding.

Maybe "progress could be accelerated greatly with a funded initiative of the kind we describe"?

File: latex/Research_Strategy.tex [hunk 2]
```diff
@@ -356,10 +356,10 @@
...
 A funded, coordinated effort allows the targeted collection of datasets designed to focus on the most important problems. 
 With multiple investigators and collaborators, we are poised to respond and adapt to new challenges and opportunities which emerge in a manner not previously possible.
-Additionally, we will be able to continue experimental work until the necessary data is collected rather than terminating it at a specific time point dictated by industry funding -- allowing us to do things like ensure the full dynamic range of log~D values is covered, unlike in SAMPL5~\cite{rustenburg_measuring_2016, bannan_blind_2016}.
+Additionally, we will be able to continue experimental work until the necessary data is collected rather than terminating it at a specific time point dictated by industry internships -- allowing us to do things like ensure the full dynamic range of log~D values is covered, unlike in SAMPL5~\cite{rustenburg_measuring_2016, bannan_blind_2016}.
```
Comment (jchodera) [inline latex/Research_Strategy.tex]: These separators should be em-dashes (`---`)

File: latex/Research_Strategy.tex [hunk 3]
```diff
@@ -458,8 +458,9 @@
...
-Thus, we believe that host-guest binding provides a critical step towards accurately modeling biomolecular interactions, focusing the field on issues not commonly encountered in physical property challenges (such as the importance of accurately modeling ionic conditions) that are highly relevant for protein-ligand interactions.
+That is, binding in host-guest systems introduces a wider variety of challenges relevant to biomolecular interactions, but without the full array of challenges seen in protein-ligand interactions, as reviewed recently~\cite{mobley_predicting_2016}.
+Thus, new data for SAMPL challenges on host-guest binding is critical to provide challenges of intermediate complexity between those of physical properties and those in biomolecular binding.
+We believe that host-guest binding provides a vital step towards accurately modeling biomolecular interactions, focusing the field on issues not commonly encountered in physical property challenges (such as the importance of accurately modeling ionic conditions) that are highly relevant for protein-ligand interactions.
```
Comment (jchodera) [inline latex/Research_Strategy.tex]: > We believe that host-guest binding provides 

"Host-guest binding challenges provide"

File: latex/Research_Strategy.tex [hunk 4]
```diff
@@ -701,6 +702,7 @@
 During the off years, SAMPL challenges will still run, but discussion of and dissemination of results will be via asynchronous means (as discussed below) and a ``virtual workshop'' consisting of talks and interaction over Google Hangouts or YouTube Live.
 	%     b. Coordinating challenges with them, submission deadlines offset from D3R challenges
 While coordination with D3R will mostly be at the level of workshops, we will also ensure that SAMPL challenge submission deadlines are offset from D3R deadlines to allow maximum community participation in both efforts.
+If the D3R effort is not renewed beyond its current funding cycle, we will run SAMPL workshops independently, controlling costs via the model we use for our Workshops on Free Energy Methods in Drug Design -- specifically, most participants will pay their own way to the workshop, and we will seek pharmaceutical and software industry sponsorships to defray costs.
```
Comment (jchodera) [inline latex/Research_Strategy.tex]: change en-dash (`--`) to closed-up em-dash (`---`)

File: latex/Research_Strategy.tex (hunk 1 above)
Comment (davidlmobley) [reply latex/Research_Strategy.tex]: Changing it to be something along those lines.

File: latex/Research_Strategy.tex (hunk 3 above)
Comment (davidlmobley) [reply latex/Research_Strategy.tex]: Done.

**Comment by davidlmobley** [comment]
Merging and sending off. Thanks!

**Pull request merged**